assets/chezmoi.io/docs/reference/templates/keepassxc-functions/index.md

```diff
@@ -18,3 +18,6 @@ cache it for the duration of chezmoi's execution. Setting `keepassxc.mode` to
 `open` will tell chezmoi to instead open KeePassXC's console with `keepassxc-cli
 open` followed by `keepassxc.args`. chezmoi will use this console to request
 values from KeePassXC.
+When setting the `keepassxc.mode` to `builtin`, chezmoi uses a builtin library to
+access a keepassxc database. This mode becomes handy if the keepassxc-cli is not
+available.
```

Comment:
```markdown

When setting `keepassxc.mode` to `builtin`, chezmoi uses a builtin library to
access a keepassxc database, which can be handy if `keepassxc-cli` is not
available. Some features available in KeePassXC such as Yubikey-enhanced
encryption, may not be available with builtin support.
```

Note the extra blank line at the top.

Reply:
> ```
> When setting `keepassxc.mode` to `builtin`, chezmoi uses a builtin library to
> access a keepassxc database, which can be handy if `keepassxc-cli` is not
> available. Some features available in KeePassXC such as Yubikey-enhanced
> encryption, may not be available with builtin support.
> ```
> 
> Note the extra blank line at the top.

Not sure what you mean wit that. What should be changed?

Reply:
In your original text, you had two paragraphs right next to each other. They should be separated by a blank line.

With the rest of it, I am suggesting a change.